src/screens/TestPage.js

```diff
@@ -0,0 +1,293 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import {
+  Container,
+  Content,
+  Text,
+  Card,
+  CardItem,
+  Thumbnail,
+  Button,
+  Spinner,
+  Radio
+} from "native-base";
+import * as _ from "lodash";
+import { AsyncStorage, NetInfo, FlatList, View } from "react-native";
+import { Row, Col, Grid } from "react-native-easy-grid";
+import styles from "../styles";
+import CustomButton from "../components/CustomButton";
+import HorizontalLine from "../components/HorizontalLine";
+import { callingHelp } from "../actions";
+import { notify } from "../helper/notify";
+import { COLOR } from "../styles/color";
+import TimerCountdown from "react-native-timer-countdown";
+
+class TestPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      timer: null,
+      count: 0,
+      question: [],
+      solution: [],
+      isSubmit: false,
+      isLoading: false,
+      isOnline: true,
+      show: false,
+      selected: false
+    };
+    this.handleNetwork = this.handleNetwork.bind(this);
+  }
+  componentDidMount() {
+    AsyncStorage.getItem("question", (err, result) => {
+      if (result !== null) {
+        const question = JSON.parse(result);
+        this.setState({ question: question.data, count: question.data.count });
+      }
+    });
+    NetInfo.isConnected.addEventListener(
+      "connectionChange",
+      this.handleNetwork
+    );
+  }
+  handleNetwork(isconnect) {
+    //functinality for net connection at time of answering paper
+    this.setState({ isOnline: isconnect });
+    // this.state.isOnline ? this.setState({ show: false }) : null;
+  }
+  componentWillUnmount() {
+    NetInfo.isConnected.removeEventListener(
+      "connectionChange",
+      this.handleNetwork
+    );
+  }
+
+  static navigationOptions = ({ navigation }) => {
+    const name = navigation.getParam("name");
+    const profile_pic = navigation.getParam("profile_pic");
+    const counter = navigation.state.params.data.timeForExam * 60 * 1000;
+    return {
+      title: name,
+      headerLeft: (
+        <Content padder>
+          <Thumbnail small source={{ uri: profile_pic }} />
+        </Content>
+      ),
+      headerRight: (
+        <Text style={styles.text}>
+          Remaining Time :{" "}
+          <Text style={{ color: COLOR.Red }}>
+            {/* {this.state.counter} */}
+            <TimerCountdown
+              initialSecondsRemaining={counter}
+              onTick={() => {}}
+              onTimeElapsed={() => {}}
+              allowFontScaling={true}
+              style={{ fontSize: 15 }}
+            />
+          </Text>
+        </Text>
+      )
+    };
+  };
+
+  handleCallHelp = async () => {
```

Comment:
can we create a seperate component for call for help and put all code there for it